README.md

```diff
@@ -8,216 +8,85 @@ that simplifies provisioning, build and release of multi-tiered
+5. Add your **Credentials Type**:
+    -   **Username and Password**: CloudBees CD username and passport for Connection
+
+    - **Stored Credential**: Used to configure authentication via stored credentials, which can be username/password or secret text, which supports an SSO session ID token
+    > **_TIP:_**  To configure a connection using an SSO session ID token, refer to [Configuring an SSO session ID token for CloudBees CD](#configuring-an-sso-session-id-token-for-cloudbees-cd).
+
+6. Both **Do not send build details to CloudBees CD** and **Override CloudBees CD SSL Validation Check** are **optional** settings. You can select the **?** icon for both and read the description to decide if you want to use these features.   
+7. Select **Test Connection** to ensure your credential is working correctly. If you receive a ``Success`` message, your configuration is ready to use. If you receive an error code, ensure your **Server URL** is correct. If it is, typically there was an error in the credential configuration and the configuration should be reconfigured.
+
+## Configuring an SSO session ID token for CloudBees CD
+The CloudBees CD plugin allows you to authenticate actions using an SSO session ID token. Before starting:
+- You must have a generated CloudBees SSO session ID token. If you have not generated a CloudBees SSO session ID token, refer to [Generate an SSO session ID token](https://docs.cloudbees.com/docs/cloudbees-cd/latest/intro/sign-in-cd#_generate_a_sso_session_id_token) for help.
+- You must have already set up a **CloudBees CD** configuration and saved it to Jenkins. If not, follow the steps in [Connecting to your CloudBees CD server](#connecting-to-your-cloudbees-cd-server) until **Credentials Type**. 
+
+The following steps will help you configure a CloudBees SSO session ID token within a CloudBees CD configuration.
+
+1. Navigate to **Manage Jenkins** > **Configure System** and under **CloudBees CD** > **Configurations**, and find the configuration you want to add an SSO session ID token to. 
+2. Under **Credentials Type**, select **Stored Credential** and a new entry field will appear.
+3. For **Stored Credential**, select **Add** and **Jenkins**. A new window should appear for the **Jenkins Credential Provider**.
+4. By default, the **Domain** field is set to *Global credentials (unrestricted)* and is unchangeable. 
+5. For **Kind**, select **Secret text**. 
+6. Select the **Scope** you want to use your token for. 
+7. In the **Secret** field, enter your CloudBees SSO session ID token. 
+8. The **ID** field is **optional**. If you want to provide and **ID**, select the **?** icon and read the message to ensure you understand the purpose and requirements of this field.
+9. The **Description** field is **optional**. However, it is suggested to provide one  to help you keep track of the token configuration as it used as the **Credentials** > **Name** in your profile.
```

Comment:
```suggestion
9. The **Description** field is **optional**. However, it is suggested to provide one to help you keep track of the token configuration because it is used as the **Credentials** > **Name** in your profile.
```